gc/base/GCExtensionsBase.hpp

```diff
@@ -406,7 +407,7 @@ class MM_GCExtensionsBase : public MM_BaseVirtual {
 
 	uintptr_t workpacketCount; /**< this value is ONLY set if -Xgcworkpackets is specified - otherwise the workpacket count is determined heuristically */
 	uintptr_t packetListSplit; /**< the number of ways to split packet lists, set by -XXgc:packetListLockSplit=, or determined heuristically based on the number of GC threads */
-
+	bool packetListSplitForced;
```

Comment:
add a comment that this is a flag to be used to distinguish if the value is externally forced (by for example command line option) or it's set to a default value